### PR TITLE
First version

### PR DESCRIPTION
**Pull request by jesusfcr**
Composite action to add vulcan-local scanning to GitHub repositories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,95 @@
...
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Scan with vulcan-local.
+        uses: adevinta/vulcan-local-action@master
+```
+
+Scans local repository with default checks:
+
+- Perform static analusis over the current repository. See  `scan-repo` option.
```
Comment (manelmontilla) [inline README.md:28]: s/analusis/analysis

File: action.yaml [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    options:
+      - CRITICAL
+      - HIGH
+      - MEDIUM
+      - LOW
+      - INFO
+      - NEVER
+
+  scan-repo:
+    description: Wether to scan the local repo path
```
Comment (danfaizer) [inline action.yaml:56]: /Wether/Whether

File: action.yaml [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    required: false
+    default: vulcan.yaml
+
+  use-local-config:
+    description: Wether to use local-config file if exists.
+    type: boolean
+    required: false
+    default: true
+
+  log-level:
```
Comment (danfaizer) [inline action.yaml:19]: Is possible to specify the default? (even as a comment)

File: README.md [hunk 4]
```diff
@@ -1 +1,95 @@
...
+- Applies configs in `vulcan.yaml` if they exists in the root of the repo. See `local-config` and `use-local-config` options..
```
Comment (manelmontilla) [inline README.md:29]: s/exists/exist
s/options../options. ?

**Review by manelmontilla — APPROVED** [review]
Minor typos that can be addressed in future PRs

**Review by danfaizer — CHANGES_REQUESTED** [review]
Added a couple of minor comments.
Great job mate!

File: README.md (hunk 1 above)
Comment (jesusfcr) [reply README.md:28]: Thanks

File: README.md (hunk 4 above)
Comment (jesusfcr) [reply README.md:29]: thanks

File: action.yaml (hunk 2 above)
Comment (jesusfcr) [reply action.yaml:56]: thanks

**Review by manelmontilla — APPROVED** [review]
LGTM

**Review by danfaizer — APPROVED** [review]
Great work! :clap:

**Pull request merged**